### ISSUE TITLE
Map visual bug

### ISSUE THREAD

**Issue by Huntender** [SunwellTracker/issues#2452]
Decription: It happened to me recently in Wintergrasp battle but it happened several times in the past too that I see one of my party/raid members to be still in Wintergrasp (not moving) but running on the place like when he's lagging, but he's actually in another place. Its wrong position is reflected on the map dot too.

How it works: Rarely, and mostly in Wintergrasp, when I go to his location a party/raid member is displayed to be not moving either on the map and in the game (the character is running on place like with lag), but in reality he's moving in a different place of the map. In those cases maybe he sees me to be bugged in the same manner. This bug disappears when one player communicates the position to the other and once they meet the position on the map is fixed, and probably the "fake image" on the map disappears. Additional info: 1) probably both players must be moving 2) it usually happened when I came from dalaran (right after a login?) then joined the WG raid or a party.

How it should work: No visual fittitous images should appear in Wintergrasp.

Source (you should point out proofs of your report, please give us some source): I'm sorry I cannot bring any proof but I hope you can trace back the issue. I think it happened mostly (only?) within the Wintergrasp perimeter. Whenever happened it brought confusion because Whenever I went on place of the party member, he wasn't really in that spot. It is quite rare but a bit annoying.

**Comment by Azathothh** [comment]
"Proof below"
I don't see any link to any proofs :P 
 I've tried to do this : 
"char A invites char B
char B disbandes the pary
char C invites char A"
and everything was fine with map and locations of players on it. Disable all addons and try again, as it may be addon issue. 
Also it may be completely not related to your steps to reproduce - it may be related to location where you doing it and different phasing for different characters on certain locations

**Comment by Huntender** [comment]
Hello again. I've tested this issue further and discovered that this happens only on 2 characters of a single account. I saw that both the characters had few quests in common and i'm pretty sure one of them is causing this bug and bugging the client. I've tried 2 clients, both without addons.
Once I've deleted these quests on one characters and restarted the client, the bug didn't occur anymore, but it's still occurring on the other character which still has the quests. 
The quests i've deleted are:
Defend the siege, The broken front, Blood and Sand, The battle for Arathi Basin. Between these, I think either defend the siege or the battle for arathi basin are causing this issue. I hope it makes sense and that i didn't do mistakes...
BTW the test this time was:
bugged chars- one in Dalaran and the other one in Icecrown, 
not bugged char for testing- flying in wintergrasp to check if its position was refreshing

Imma test those 2 quests then report back here

**Comment by Huntender** [comment]
Nope, even deleting the quests didn't work, and I also have restarted both the clients. I'm trying to figure out which character of the three is causing the issue

**Comment by Huntender** [comment]
OK. I'm pretty sure I have the answer... I think it's something related with the RAF pairing, because the bug only seems to occur between 2 characters that are binded together with the RAF bonus. it happens to the pair of 80s i was testing before and also to a pair of low lvls, so it isn't about the quests. Basically it disrupts the map position and possibly it is this that creates the fake image in one place.

**Comment by Huntender** [comment]
@Azathothh Hello, just making sure you didn't miss the latest update of this thread. Basically it seems to be an issue bound to raf system so your testing would be very important. thank you!
